raft/src/main/java/org/apache/kafka/snapshot/SnapshotWriter.java

```diff
@@ -113,10 +113,8 @@ private void finalizeSnapshotWithFooter() {
      * Create an instance of this class and initialize
      * the underlying snapshot with {@link SnapshotHeaderRecord}
      *
-     * @param snapshot a lambda to create the low level snapshot writer
      * @param maxBatchSize the maximum size in byte for a batch
      * @param memoryPool the memory pool for buffer allocation
-     * @param time the clock implementation
```

Comment:
Should this be `snapshotTime` instead of `time`?

Reply:
You're right! I just checked the parameter, and make sure there's no such parameter called `time`. But, yes, it's put the wrong name. Thanks for the review!